### ISSUE TITLE
Crashing on atmosphere 19.3

### ISSUE THREAD

**Issue by AykoPredator** [spacemeowx2/ldn_mitm#94]
i know you haven't updated this but i was just making this to pretty much let you know i am getting a panic on atmopshere 19,3, thank you in advanced

**Comment by MNaikSwalayan** [comment]
Second that, I missclick the update firmware prompt, and now it just crash everytime I tried to go lan play.

**Comment by RLSnake109** [comment]
pleaseee update the Idn_mitm kip

**Comment by vkaziv** [comment]
I need local play

**Comment by Shinusagi** [comment]
I think we will need to wait longer sadly, let's hope they release a new version soonish

**Comment by Aeuko** [comment]
Hope to update soon， Monster Hunter Rise 3.0 is ready to go

**Comment by yanisdreemurr** [comment]
people who insist while spacemeowx2 is trying to recode the program, just be patient and stop creating "Issues" while others are already asking the question, and yes it is normal that you have a fatal error or whatever.

**Comment by emodeshengge** [comment]
Looking forward to the update of the new version0.19.3(｡･ω･｡)ﾉ♡

**Comment by AuroraWright** [comment]
Try this, it's just a rebuild with latest libstratosphere (no other code changes)
[ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)

**Comment by MNaikSwalayan** [comment]
> Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)

Thank you. I tried your build and it works well.

though, I'm curious how do you rebuild the code? If you don't mind, may you explain it? 
 
ps: I tried to clone ldn_mitm repository to my private repository and I also update atmosphere_libs. After that, I run "Build" workflow on "action" tab to rebuild ldn_mitm but it always fails at "Run spacemeowx2/devkitpro-action@latest".

**Comment by yanisdreemurr** [comment]
> Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)

I tested your build and it works fine on my side, thank you so much @AuroraWright

**Comment by vkaziv** [comment]
I tested the build. Don't crash looking for lan but here the problem I can't find a room to join. So I created a room and on my other switch I can't find the room on the switch that's having these issue other switch is a non hacked lite. On my hacked switch I go into homebrew and go into mitm I turn off mitm go back into a game do local and I can find the non hack switch as well on my non hack I can find the hack switch but when trying to join gives a communication error so not to sure what's going on but figure it will help someone in the future

**Comment by Shinusagi** [comment]
> Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)

Thanks a lot! Worked fine thus far!

**Comment by vamidi** [comment]
> > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> 
> Thanks a lot! Worked fine thus far!

Are you able to play with people? I see in all the previous messages that the rooms dont work.

**Comment by Shinusagi** [comment]
> > > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> > 
> > 
> > Thanks a lot! Worked fine thus far!
> 
> Are you able to play with people? I see in all the previous messages that the rooms dont work.

Yes, at least for me it worked and I even was appearing (as well my friend) in the LanPlay Lobbies list, for Atmosphere 19.3 it is working, not sure about other previous versions, also that specific zip file, not the last release from April in the download section

**Comment by vkaziv** [comment]
> > > > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > > > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> > > 
> > > 
> > > Thanks a lot! Worked fine thus far!
> > 
> > 
> > Are you able to play with people? I see in all the previous messages that the rooms dont work.
> 
> Yes, at least for me it worked and I even was appearing (as well my friend) in the LanPlay Lobbies list, for Atmosphere 19.3 it is working, not sure about other previous versions, also that specific zip file, not the last release from April in the download section

Not fully working because I can't find lobbies on lan play I'm also on latest ams

**Comment by MNaikSwalayan** [comment]
> 
> 
> > > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> > 
> > 
> > Thanks a lot! Worked fine thus far!
> 
> Are you able to play with people? I see in all the previous messages that the rooms dont work.

In my case, it works well. I can see lobbiest, join it, also play for sometime(still disconnected occasionally because of latency).

**Comment by AuroraWright** [comment]
Fwiw the build I posted should also work fine on 0.19.4.

**Comment by herve-ch** [comment]
> 
> 
> > > > > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > > > > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> > > > 
> > > > 
> > > > Thanks a lot! Worked fine thus far!
> > > 
> > > 
> > > Are you able to play with people? I see in all the previous messages that the rooms dont work.
> > 
> > 
> > Yes, at least for me it worked and I even was appearing (as well my friend) in the LanPlay Lobbies list, for Atmosphere 19.3 it is working, not sure about other previous versions, also that specific zip file, not the last release from April in the download section
> 
> Not fully working because I can't find lobbies on lan play I'm also on latest ams

It is woking perfectly thanks for the new build Aurora

**Comment by Glazelf** [comment]
> Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)

Doesn't seem to work for me on fw 12.0.3 | ams 19.4. Is there any chance you could compile it for 19.4? I'm not sure if there is any difference as far as ldn_mitm compilation goes.

**Comment by herve-ch** [comment]
> > Try this, it's just a rebuild with latest libstratosphere (no other code changes)
> > [ldn_mitm_0.19.3.zip](https://github.com/spacemeowx2/ldn_mitm/files/6601908/ldn_mitm_0.19.3.zip)
> 
> Doesn't seem to work for me on fw 12.0.3 | ams 19.4. Is there any chance you could compile it for 19.4? I'm not sure if there is any difference as far as ldn_mitm compilation goes.
> 
> Behaviour:
> Config opens correctly and no crashes whatsoever, but seems to function as if ldn wasn't installed. [Log info](https://pastebin.com/mStNUAsB)

It is working fine for me on atmosphere 19.4 last firmware

**Comment by spacemeowx2** [comment]
Can you try this?

 https://github.com/spacemeowx2/ldn_mitm/actions/runs/962603993

**Comment by Glazelf** [comment]
> Can you try this?
> 
> https://github.com/spacemeowx2/ldn_mitm/actions/runs/962603993

Thanks for the reply and the help! Behaviour seems to be unchanged though, sadly. Here's the log: [ldn_mitm.log](https://github.com/spacemeowx2/ldn_mitm/files/6700750/ldn_mitm.log).

**Comment by apeli11** [comment]
Hi, I'm having the same issue with ams 19.4 . I've tried with all the builds in this thread but I still can not join a game, an error of unstable conection appears   (┬┬﹏┬┬)   Hope you can help me, I would really appreciate it guys  (｡･ω･｡)

**Comment by bettydreemurr** [comment]
> Can you try this?
> 
> https://github.com/spacemeowx2/ldn_mitm/actions/runs/962603993

i just tested this build, and it still works great for me, thanks a lot for your work!

**Comment by vamidi** [comment]
> Can you try this?
https://github.com/spacemeowx2/ldn_mitm/actions/runs/962603993

Lots of error connecting to friends.  I can setup a host, but when I start playing it all disconnects.
